### PR TITLE
Add Calc test

### PR DESCRIPTION
**Pull request by Dawn-githup**
pull Calc testing!

### CODE REVIEW

File: contracts/test/Calc.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  describe("Calc contract", function () {
+    it("Deployment computing contract", async function () {
+      const [owner] = await ethers.getSigners();
+  
+      const Calc = await ethers.getContractFactory("Calc");
+  
+      const hardhatCalc = await Calc.deploy();
+  
+      const addBalance = await hardhatCalc.add(5,6);
+      const sumBalance = await hardhatCalc.sub(5,6);
```
Comment (Flouse) [inline contracts/test/Calc.js]: Please use the `expect` function to check the result.

File: contracts/test/Calc.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    });
+  });
```
Comment (Flouse) [inline contracts/test/Calc.js]: Add new line at the end of file

**Comment by Flouse** [comment]
Please format the code style.

**Comment by Dawn-githup** [comment]
format the Calc test

**Review by keroro520 — COMMENTED** [review]
The `contracts/package-lock.json` update so many lines. What happens to package-lock.json?

**Review by Flouse — CHANGES_REQUESTED** [review]

**Comment by Dawn-githup** [comment]
> `contracts/package-lock.json`更新这么多行。package-lock.json 会发生什么？
'contracts/package-lock.json' changes do not need to update dependencies. I will delete this lock

**Pull request closed without merging**